include/clang/AST/Decl.h

```diff
@@ -1584,8 +1582,6 @@ class FunctionDecl : public DeclaratorDecl, public DeclContext,
   /// 'enum Y' in 'void f(enum Y {AA} x) {}'.
   ArrayRef<NamedDecl *> DeclsInPrototypeScope;
 
-  BoundsExpr *ReturnBoundsExpr;
```

Comment:
There are already comments on the getter functions in the superclass of FunctionDecl.  These will be picked up the automatic doc generation and by IDE tool tips (for IDEs that understand /brief comments).  I've add a comment to the setter function too.